packages/plugin-html/src/HtmlNode.ts

```diff
@@ -0,0 +1,13 @@
+import { AtomicNode } from '../../core/src/VNodes/AtomicNode';
+
+export interface HtmlNodeParams {
+    domNode: Node;
+}
+
+export class HtmlNode extends AtomicNode {
+    domNode: Node;
+    constructor(params: HtmlNodeParams) {
+        super();
+        this.domNode = params.domNode;
+    }
+}
```

Comment:
I suppose it's to have a `VNode` in our tree that represents a dom Node that needs to specifically be the exact same dom Node at rendering rather than being re-generated, to keep the listeners etc etc. If that's the case then I guess the name of the class could better reflect this.